examples/food-imports/src/index.md

```diff
@@ -74,6 +91,8 @@ const nest = {
     children: Array.from(children, ([name, value]) => ({name, value}))
   }))
 };
+// reorder nodes to match category order established above
+nest.children = d3.sort(nest.children, (a,b) => categories.indexOf(a.name) - categories.indexOf(b.name))
```

Comment:
```suggestion
nest.children = d3.sort(nest.children, (d) => categories.indexOf(d.name))
```

d3.sort accepts an accessor, which is often simpler than using a comparator. You can inline this in L74.